tests/ui/specs/Popup/Popup-specs.js

```diff
@@ -970,8 +989,8 @@ describe('Popup', function () {
 				Page.waitForOpen(popup);
```

Comment:
At line 921, I think we should remove the expect since it's pointer mode. I got a fail. WDYT?

Reply:
We don't get a fail here. Could you explain please why should I remove expect?  Thank you!

Reply:
I left this comment here because I couldn't write comment at line 921.
https://github.com/enactjs/agate/pull/503/files/acfceb4310bbf2d06cd73e42bd049a363e0d3fe2#diff-9f14c06315ab304e32519ce369483b6a5a320c57b7a0aa59a19adcaff261e361R921
Since it's pointer mode, it shouldn't have focus on the popup button again.

Reply:
Oh, yes, now I found the line 😄  Thank you!